### DISCUSSION TITLE
Citi中国

### DISCUSSION THREAD

**Discussion by tonyqus** [shnug/jobs#45] [General]
首先，花旗国内已经基本不招.net开发了，后端都是Java。headcount已经freeze了2年了，现在基本不招，听说近期还可能裁员（还没确认）。

目前是hybrid，可以2-3在家办公。基本不加班，不忙。

虽然也是敏捷开发，每天有standup meeting，但据说比较混乱，每个Sprint可能会随机插入任务，且没有Scrum master负责把控任务量，甚至连Burnchart都画不出。

坊间传，花旗已经撤离中国，实际上只是把个人业务转卖了，并没有关闭。

**Comment by GGJason** [comment]
有開都好，有些公司已經head count freeze 大半年了

**Reply by tonyqus** [reply]
你在台湾，你是怎么知道的？

**Reply by GGJason** [reply]
我來香港啦哈哈，我公司也是Headcount freeze 搞了我一陣
